_layouts/post.html

```diff
@@ -23,14 +23,6 @@ <h1 itemprop="name headline">{{ page.title }}</h1>
     </div>
   </header>
 
-  <section class="section-padding bg-grey" itemprop="articleBody">
```

Comment:
Did you run a build? This is the code for the blog post to display, but it has been removed so this needs to be reverted.

Reply:
Changes were reverted.